impl/src/main/java/org/jboss/weld/executor/AbstractExecutorServices.java

```diff
@@ -132,4 +133,15 @@ protected void shutdown() {
             Thread.currentThread().interrupt();
         }
     }
+
+    /**
+     * This method is invoked with the body of invokeAllAndCheckForExceptions(Collection<? extends Callable<T>> tasks).
+     *
+     * It allows to wrap the tasks with some additional logic. For instance, Weld's CommonForkJoinPoolExecutorServices
```

Comment:
It's always better to link classes by means of `{@link}`... and make refactoring possible ;-)